docs/mr_proc/workflow/dicom_org.md

```diff
@@ -2,40 +2,49 @@
+
+2. Download DICOM dumps (e.g. ZIPs / tarballs) in the `<DATASET_ROOT>/downloads` directory. Different visits (i.e. sessions) must be downloaded in separate sub-directories and ideally named as listed in the `global_config.json`. The DICOM download and extraction process is highly dataset-dependent, and we recommend using custom scripts to automate it as much as possible.
+3. Extract (and rename if needed) all participants into `<DATASET_ROOT>/scratch/raw_dicoms` separately for each visit (i.e. session). 
+    - At this point, the `doughnut.csv` should have been updated to reflect the new downloads (`downloaded` column set to `True` where appropriate). We recommend doing this in the download script (i.e. in Step 2), but `workflow/dicom_org/check_dicom_status.py` can also be run with the `--regenerate` flag to search for the expected files (this can be very slow!).
+
 
 !!! note
 
-    **IMPORTANT**: the participant-level directory names should match participant_ids in the mr_proc_manifest.csv. It is recommended to use participant_id naming format to exclude any non-alphanumeric chacaters (e.g. "-" or "_"). If your participant_id does contain these characters, it is still recommended to remove them from the participant-level DICOM directory names (e.g., QPN_001 --> QPN001).  
+    **IMPORTANT**: the participant-level directory names should match `participant_id`s in the `mr_proc_manifest.csv`. It is recommended to use `participant_id` naming format to exclude any non-alphanumeric chacaters (e.g. "-" or "_"). If your participant_id does contain these characters, it is still recommended to remove them from the participant-level DICOM directory names (e.g., QPN_001 --> QPN001).
 
 !!! note
 
     It is **okay** for the participant directory to have messy internal subdir tree with DICOMs from multiple modalities. (See [data org schematic](data_org.md) for details). The run script will search and validate all available DICOM files automatically. 
 
 
-- Run [run_dicom_org.py](https://github.com/neurodatascience/mr_proc/blob/main/workflow/dicom_org/run_dicom_org.py) to:
+4. Run [`run_dicom_org.py`](https://github.com/neurodatascience/mr_proc/blob/main/workflow/dicom_org/run_dicom_org.py) to:
     - Search: Find all the DICOMs inside the participant directory. 
     - Validate: Excludes certain individual dicom files that are invalid or contain scanner-derived data not compatible with BIDS conversion.
-    -  Copy or Symlink (**Recommended**): Creates symlinks from `raw_dicoms/` to the `<DATASET_ROOT>/dicom`, where all participant specific dicoms are in a flat list.
+    - Symlink (default) or copy: Creates symlinks from `raw_dicoms/` to the `<DATASET_ROOT>/dicom`, where all participant specific dicoms are in a flat list. The symlinks are relative so that they are preserved in containers.
+    - Update status: set the `organized` column to `True` in `doughnut.csv`.
```

Comment:
I would slightly change wording to say "if successful, updates the status of `organized` column to `True` in `doughnut.csv`